.github/workflows/flake8.yml

```diff
@@ -0,0 +1,39 @@
+name: Flake8 check
+
+on:
+  push:
+    branches:
+    - main
+    
+    
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.9
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+    - name: Install Poetry
+      uses: snok/install-poetry@v1
+
```

Comment:
Why do we need poetry here? The only dependency we need is flake8 + plugins right? Because with poetry we also install delta, pytest etc. which slows down the pipeline. Shouldn't "py-actions/flake8@v2" already do the job?

Reply:
To explain my reasoning, I would expect this pipeline to be the same as the black one, just with "py-actions/flake8@v2"

Reply:
My bad @robertkossendey I thought I committed the correct version of the pipeline. Yes the correct version that I pushed just now has only the required YAML.

Reply:
Great, then the only nitpick I have is removing flake8 again from the pyproject.toml and revert the poetry.lock! :)

Reply:
@robertkossendey reverted them to the original state